packages/common-ui-web/src/components/VehicleDynamicWireframe/VehicleDynamicWireframe.tsx

```diff
@@ -68,6 +67,10 @@ function createGetAttributesCallback(
 }
 
 function getVehicleModel(vehicleType: VehicleType): VehicleModel {
+  if (vehicleType === VehicleType.SUV) {
+    // eslint-disable-next-line no-param-reassign
+    vehicleType = VehicleType.CUV;
+  }
```

Comment:
I prefer this : 

```ts
cons type = vehicleType === VehicleType.SUV ? VehicleType.CUV : vehicleType;
```

over this : 

```ts
if (vehicleType === VehicleType.SUV) {
  // eslint-disable-next-line no-param-reassign
  vehicleType = VehicleType.CUV;
}
```

Reply:
This would be string forward and don't need to create a new variable. because vehicleType is good variable name.